src/server/api/routers/auth.ts

```
@@ -224,6 +224,7 @@ export const authRouter = createTRPCRouter({
       });
 
       const isDev = env.NODE_ENV === 'development';
+      logger.info("requestMobileOtp", { isDev, NODE_ENV: env.NODE_ENV, email: normalizedEmail });
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'email' should be before 'NODE_ENV'.

(sort-keys-fix/sort-keys-fix)